sdk/src/feature_set.rs

```diff
@@ -75,6 +75,10 @@ pub mod bpf_loader_upgradeable_program {
     solana_sdk::declare_id!("FbhK8HN9qvNHvJcoFVHAEUCNkagHvu7DTWzdnLuVQ5u4");
 }
 
+pub mod set_upgrade_authority_via_cpi_enabled {
```

Comment:
These aren't ordered in any way, it's just been convention to add it to the end

Reply:
Moved to the end